muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/guider/body/application.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import {connect, Dispatch} from 'react-redux';
-
-import ApplicationPanel from '~/components/general/application-panel';
+//import ApplicationPanel from '~/components/general/application-panel';
```

Comment:
If you moved it from one place to the other, and the previous general file is removed, then just remove the line